app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,34 @@
+.board {
+ 	width: 95%;
+  height: 95%;
+  border: 3px solid #333;
+  margin: 10px auto 10px auto;
+  vertical-align: middle;
+}
+body {
+ 	padding-top: 10%; 
+ 	padding-bottom: 10%;
+}
+td {
+  empty-cells: show;
+}
+table.one, .board {
+  border-collapse: collapse;
+ 	height: 45em;
+ 	width: 45em;
```

Comment:
Right here, you're assigning a height and width of 45em to the `.board` div, but on lines 2 and 3, you're setting those properties equal to 95%. Only one set of properties will be applied (the one that comes last), so you shouldn't leave them both in the code.

This leads me to ask why you're still wrapping the table in a `board` div. It doesn't give you anything that can't be accomplished by styling the table directly and just causes problems, which you've managed to deal with right now, but which may crop up again when we make the board responsive. Ultimately, it's up to you, and I won't hold up the merge because of this choice, but I think it'll just cause unnecessary trouble.